spec/expression.dd

```diff
@@ -1865,14 +1865,14 @@ $(GNAME ParameterWithMemberAttributes):
 
 $(GNAME FunctionLiteralBody2):
     $(D =>) $(GLINK AssignExpression)
-    $(GLINK2 function, SpecifiedFunctionBody)
+    $(GLINK2 statement, BlockStatement)
```

Comment:
But this one doesn't look right, this is allowed:
```D
auto b = function int(int x) do {return true;};
```

Reply:
Sorry, restored this line.